src/components/RouteInputForm/RouteInputForm.jsx

```diff
@@ -19,13 +21,23 @@ const RouteInputForm = ({ onStartingPointChange, onDestinationChange }) => {
           ariaLabelledBy="destination-label"
         />
       </div>
+      <div>
+        <label id="arrive-by-label">Arrive By</label>
+        <DateTimeInput
+          value={arriveByValue}
+          onChange={onArriveByDateTimeChanged}
+          ariaLabelledBy="arrive-by-label"
```

Comment:
You are passing along this property but don't do anything with it in `DateTimeInput`. No need to implement accessibility for now. Let's remove it. We'll implement it later when we actually need it for interaction tests.